doc/publishing-open-data-in-fiware/how-to-offer-datasets-including-context-information-through-the-wstore/offering-datasets-directly-through-the-ckan-interface.md

```diff
@@ -1,15 +1,44 @@
+the Store Publisher plugin allows FIWARE users to offer their published
+datasets in the FIWARE Store, by enabling the creation of offerings directly
+from the CKAN interface. In this way, users do not need to go to a different
+system saving time and improving the usability.
+
+In order to create an offering using the CKAN interface, you must be logged
+in, so you can manage your published datasets. Once you have entered in CKAN,
+open the dataset you want to offer and click on the “Manage” button.
 
 [![OfferingDatasetsDirectlyThrough1](../images/OfferingDatasetsDirectlyThrough1.png)](../images/OfferingDatasetsDirectlyThrough1.png)
 
-Then, you should select the “Publish” tab. In this tab you will be asked
-to provide some information of the offering (the name, the
-description...). If you want to publish a private dataset, you will be
-asked for a price. We remind you that if you want to create a free
-offering, you can set the “Price” field to zero. Once that you have
-filled in the fields, you have to click the “Publish Offering” button to
-finish the process.
+The displayed view, allows you to perform some management tasks like updating
+the dataset info, updating the provided data, or removing the dataset. To publish
+your dataset in the FIWARE Store select the “Publish” tab.
+
+In this tab, you have to provide some information of the offering:
+
+   - Name: Name you want to give to your offering in the FIWARE Store
+   - Description (optional): Textual description of your offering that will be displayed to FIWARE Store users
+   - Categories (optional): Set of categories of your offering. The available categories are retrieved from the FIWARE Store so you can choose among them
+   - Catalog: Catalog where you want to publish your offering. The list of catalogs where you have publish rights is retrieved from the FIWARE Store
+   - Version: Version of your offering
+   - License (optional): License of the data being published
+   - License Description (optional): Description of the license of the data
+   - Image (optional): Image to be displayed in the FIWARE Store attached to your offering. If you do not include any, the CKAN logo will be used
+   - Price (optional): Price of the offering. If you do not include this field, the offering is considered free
+
+Note that you can use the “License” and “License Description” fields in order
+to include the terms and conditions for using the data. In this regard,
+if you include those fields, customers will need to explicitly accept them
+to be able to acquire the offering.
 
 [![Offering Datasets Directly Through the CKAN
 Interface1](../images/Offering-Datasets-Directly-Through-the-CKAN-Interface1.png)](../images/Offering-Datasets-Directly-Through-the-CKAN-Interface1.png)
+
+You may have noticed that this way of creating offerings has some restrictions.
+On the one hand, the offering is created just with the dataset, so you cannot
```

Comment:
On the one hand .. On the other hand, is a correct structure
http://dictionary.cambridge.org/es/diccionario/ingles/on-the-one-hand-on-the-other-hand

"On one hand" is correct, but is an alternative form, not the preferred one
https://jakubmarian.com/on-the-one-hand-on-one-hand/